internal/ctxval/edge_id.go

```diff
@@ -0,0 +1,17 @@
+package ctxval
+
+import "context"
+
+// EdgeRequestId returns request id or an empty string when not set.
+func EdgeRequestId(ctx context.Context) string {
```

Comment:
Could we pls document here that it is coming from edge gateway? :)

Reply:
Done